cli/cmd/common.go

```diff
@@ -33,12 +37,28 @@ var (
 	// ReleaseImageReplacements contains the image replacements used by contrast.
 	//go:embed assets/image-replacements.txt
 	ReleaseImageReplacements []byte
-	// DefaultCoordinatorPolicyHash is derived from the coordinator release candidate and injected at release build time.
+	// CoordinatorPolicyHashesFallback are derived from the coordinator release candidate and injected at release build time.
 	//
 	// It is intentionally left empty for dev builds.
-	DefaultCoordinatorPolicyHash = ""
+	//go:embed assets/coordinator-policy-hashes-fallback.json
+	CoordinatorPolicyHashesFallback string
```

Comment:
```suggestion
	CoordinatorPolicyHashesFallback []byte
```
That would spare you the type conversion below.